services/explorer/src/pages/Home/ChartContainer.tsx

```diff
@@ -2,14 +2,23 @@ import React from 'react';
 import { Grid } from '@mui/material';
 import { BarChart } from '@taraxa_project/taraxa-ui';
 import { theme } from '../../theme-provider';
-import {
-  getMockedDagBlocksCard,
-  getMockedPbftBlocksCard,
-} from '../../api/mocks';
+import { DagBlock, PbftBlock } from '../../models';
+// import {
```

Comment:
If we don't need these anymore, you can delete them. If you might need them to test something, then leave them commented.